### ISSUE TITLE
"media_id" inconsistency between "stories" and "story_sentences"

### ISSUE THREAD

**Issue by pypt** [mediacloud/backend#456]
Our "client" Jonas has uncovered that `media_id` is not the same between some rows in `stories` and `story_sentences` for the same `stories_id`, e.g.:

```
mediacloud=# select media_id from stories where stories_id = 340320494;
 media_id 
----------
    19404
(1 row)

mediacloud=# select distinct media_id from story_sentences where stories_id = 340320494;
 media_id 
----------
    40649
(1 row)
```

Rows from `media` for those two `media_id`s suggest that one is a duplicate of the other, as denoted by `dup_media_id`:

```
mediacloud=# select * from media where media_id in (19404, 40649);
 media_id |         url         |             name             | full_text_rss | foreign_rss_links | dup_media_id | is_not_dup | content_delay | editor_notes | public_notes | is_monitored | normalized_url  
----------+---------------------+------------------------------+---------------+-------------------+--------------+------------+---------------+--------------+--------------+--------------+-----------------
    19404 | http://www.faz.net  | faz.net                      |               | f                 |        40649 |            |               |              |              | f            | http://faz.net/
    40649 | http://www.faz.net/ | Frankfurter Allgemeine (FAZ) |               | f                 |              |            |               |              | null         | t            | http://faz.net/
(2 rows)
```

`media.dup_media_id` dates back to 2015 so it's probably some sort of an unfinished deduplication work done by someone.

I think we should just go ahead and merge all the duplicate `media` rows into one without much hesitation about "backwards compatibility", i.e. without complicating matters further by adding "media IDs that were duplicates of other media IDs but are now merged" tables and whatnot. `media` table already has `normalized_url` (possibly to be used for deduplication), `dup_media_id` (from 2015) and `is_not_dup` (never read or written) columns, so taking a fourth entity into the consideration would be just too much cognitive load.

(Possibly part of #380.)

**Comment by hroberts** [comment]
I'm sure you are right that this is the result of some sort of cleanup at some point in the far past.

The holdup on the media deduping is not the tracking table.  The holdup is understanding how, exactly, the deduping will impact our existing media sources and how to merge the sources sanely without losing stories, adding falsely attributed stories to media, or messing up our media sets (we don't want to add the entire BBC to the India set just because BBC India was in the India set).

-hal

**Comment by rahulbot** [comment]
Depends on #390

**Comment by pypt** [comment]
Efforts moved to https://github.com/mediacloud/backend/issues/799.
